veos/docker/launch.py

```diff
@@ -43,6 +43,15 @@ def __init__(self, username, password):
         super(VEOS_vm, self).__init__(username, password, disk_image=disk_image, ram=2048)
         self.num_nics = 20
         self.qemu_args.extend(["-cdrom", boot_iso, "-boot", "d"])
+        self.zerotouch_disabled = False
+        self.disable_zerotouch = False
+
+        # list of images that require us to disable zerotouch for proper function
+        requires_zerotouch_disable = ['vEOS-lab-4.27.0F.vmdk']
```

Comment:
why only on this code?

Reply:
I'm inclined to agree. Shouldn't we generally turn this off?

Is there a downside to always turning it off? Seems like the simpler thing...